discord_thread_summarizer_using_openai/discord_utils.bal

```diff
@@ -0,0 +1,55 @@
+import ballerina/time;
+import ballerina/http;
+
+const time:Seconds DISCORD_EPOCH = 1420070400000.00;
+
+http:Client discordClient = check new (DISCORD_API_URL);
```

Comment:
Make module-level variables final where possible. Helps with isolated inference when called from a service, assuming the rest of the requirements are met (e.g., isolated client class).